pygmt/datasets/load_remote_dataset.py

```diff
@@ -57,7 +57,7 @@ class GMTRemoteDataset(NamedTuple):
 datasets = {
     "earth_age": GMTRemoteDataset(
         description="EarthByte Earth seafloor crustal age",
-        units="Myr",
+        units="million years",
```

Comment:
I don't think "million years" are valid units.

Let's focus on "meters" vs "m" in this PR.

Reply:
Yes! I was unsure with the crustal age dataset. Also have a preferes to keep `Myr`.